docs/vtex-io/Storefront-Guides/concepts-1/vtex-io-documentation-building-a-product-details-page.md

```diff
@@ -232,29 +232,29 @@ To customize the `flex-layout.col#product-image` and `flex-layout.col#right-col`
 
 #### Left column
 
-The left column of the second row contains the [`product-images`](https://developers.vtex.com/docs/guides/vtex-store-components-productimages) block since this was the first `flex-layout.col` block to be declared.
+The left column of the second row contains the [`product-images`](https://developers.vtex.com/docs/guides/vtex-store-components-productimages) block because this was the first `flex-layout.col` block to be declared.
```

Comment:
```suggestion
The left column of the second row contains the [`product-images`](https://developers.vtex.com/docs/apps/vtex.store-components/productimages) block because this was the first `flex-layout.col` block to be declared.
```